CHANGELOG.md

```diff
@@ -213,6 +213,10 @@ Key points:
   NonAggregate` no longer implies `(OtherType, T): NonAggregate`.
   - With `feature = "unstable"`, `(T, OtherType): NonAggregate` is still implied.
 
+- In diesel v1.4 sql functions without arguments used the `no_arg_sql_function!` macro,
```

Comment:
I moved what I had written before to this set of key points. This will probably need to be expanded but I figured it's at least a good note.